rs/nervous_system/candid_utils/src/printing.rs

```diff
@@ -0,0 +1,10 @@
+use candid::{CandidType, IDLValue};
+
+pub fn pretty<T: CandidType>(value: T) -> Result<String, String> {
```

Comment:
Can you add a triple slash comment that elaborates? E.g.

* Why not just use `#[derive(Debug)]` ? How is this different? (again, answer this in triple slash, not just in GitHub review UI).
* Does this use multiple lines?
* Give a small (yet illustrative) example. E.g. what does an object of type ```struct Widget { id: u64; name: Option<String>; }``` look like when you send it through this function?
* Even though this CAN return Err, it seems like the situation would have to be pathological and/or unusual. E.g. maybe you can say something like "the main reason this would return Err is if the input is huge; otherwise, Ok would more or less always be returned.".